src/grafana/tasks.yaml

```diff
@@ -8,6 +8,13 @@ tasks:
             name: "app.kubernetes.io/instance=grafana"
             namespace: grafana
             condition: Ready
+      # This test is only functional on full uds-core deploys
```

Comment:
Can you explain more about this test? It fails on single package tests? 

Are we testing this functionality somewhere else?

Reply:
Single package tests do not deploy the prometheus stack (or CRDs) and we [filter out](https://github.com/defenseunicorns/uds-core/blob/podmonitor-exemption/src/pepr/operator/reconcilers/package-reconciler.ts#L60) the generation of those resources in the pepr code. So running this validation in CI would only work on the full core testing we do.

This generation functionality is unit tested in the pepr code, but no e2e test exists currently (the same is true for service monitors and netpols I believe).

Reply:
@rjferguson21 I'm going to add a new test, tacking it onto the podinfo checks (to cover both pod and service monitors).